tests/components/automation/test_init.py

```diff
@@ -11,24 +11,23 @@
 from tests.common import get_test_home_assistant, assert_setup_component
 
 
+# pylint: disable=invalid-name
 class TestAutomation(unittest.TestCase):
     """Test the event automation."""
 
-    # pylint: disable=invalid-name
-
     def setUp(self):
         """Setup things to be run when tests are started."""
         self.hass = get_test_home_assistant()
         self.hass.config.components.append('group')
         self.calls = []
 
         def record_call(service):
-            """Record call."""
+            """Helper to record calls."""
             self.calls.append(service)
 
         self.hass.services.register('test', 'automation', record_call)
```

Comment:
It's out of scope, yes calls.append would work, however, it would be better to convert it to a callback so we won't initialize the thread pool at all.